timers.js

```diff
@@ -1,4 +1,35 @@
+var pauseButton = document.getElementById("pause");
+var resetButton = document.getElementById("reset");
+var startButton = document.getElementById("start");
+var seconds = 0;
+var timerId = 0;
+/* START Button*/
+startButton.addEventListener ("click", function (){
+  console.log ("I've starting the timer");
+  if (timerId === 0) {
+    document.getElementById("timer").textContent="Time elapsed: ";
+    timerId = setInterval(updateTime, 1000);
+  }
+  });
 
+  /* RESET Button*/
+  resetButton.addEventListener ("click", function (){
+    console.log ("I've reset your timer to 0");
+    document.getElementById("timer").textContent="Stop Watch";
+    clearInterval(timerId);
+    seconds = 0;
+    //document.getElementById("timer").textContent="Stop Watch";
```

Comment:
Try not to keep code around that's been commented out, unless you really need it for reference. 

Comments should be for notes, not dead code!